### PR TITLE
community: add matrix chat

### PR DESCRIPTION
**Pull request by miri64**
As @waehlisch noticed, our Matrix chat is currently not linked on the Community page. This fixes that.

### CODE REVIEW

File: community.html [hunk 1]
```diff
@@ -65,6 +65,18 @@ <h2>Join us</h2>
            </div>
          </div>
 
+         <div class="col-12 col-md-4">
+           <div class="card border-0 d-flex h-100">
+             <div class="card-header d-flex align-items-center justify-content-center h-100 border-0">
+               <a href="https://matrix.to/#/#riot-os:matrix.org" target="_blank"><img class="card-img-top" src="{{ "assets/img/matrix_logo.png" | relative_url }}" alt="Matrix logo"></a>
```
Comment (waehlisch) [inline community.html:71]: logo is not included.

**Review by waehlisch — CHANGES_REQUESTED** [review]

File: community.html (hunk 1 above)
Comment (miri64) [reply community.html:71]: Fixed.

File: community.html [hunk 2]
```diff
@@ -65,6 +65,18 @@ <h2>Join us</h2>
...
+             </div>
+             <div class="card-body px-0">
+               <!--<h4 class="card-title">Our matrix for discussions.</h4>-->
+               <a href="https://matrix.to/#/#riot-os:matrix.org" class="btn btn-outline-primary w-100" target="_blank">Join our community chat and ad-hoc discussions</a>
```
Comment (waehlisch) [inline community.html:75]: s/Join our community chat and ad-hoc discussions/Join for ad-hoc discussions/

**Review by waehlisch — CHANGES_REQUESTED** [review]

File: community.html (hunk 2 above)
Comment (miri64) [reply community.html]:
```vimscript
s/Join our community chat and ad-hoc discussions/Join for ad-hoc discussions/<cr>
Git commit --amend<cr>
Git push --force-with-lease<cr>
```
;-)

**Review by waehlisch — APPROVED** [review]

**Comment by miri64** [comment]
Mhhh.... now the layout of the `card` rectangles slightly changed per row... And the youtube logo probably should be centered. :-/

**Comment by waehlisch** [comment]
not sure what you mean. looks good at my side.

i wouldn't center the YouTube logo.

**Comment by miri64** [comment]
This is how it looks like on my end (Firefox 92.0). Note the slightly to high hackster.io logo and the very slim Youtube logo.

image

**Comment by waehlisch** [comment]
the `mt-5` in the riot and fosstodon boxes introduce the error. maybe @leandrolanzieri knows why we added this tag. i don't remember.

**Comment by leandrolanzieri** [comment]
> the `mt-5` in the riot and fosstodon boxes introduce the error. when you remove the tag., everything looks good maybe @leandrolanzieri knows why we added this tag. i don't remember.

The top margin was introduced for the second row, I guess the idea was that the matrix looked not so cluttered. We can add it to hackster or remove it altogether. 

> i wouldn't center the YouTube logo.

Me neither.

**Comment by waehlisch** [comment]
> > the `mt-5` in the riot and fosstodon boxes introduce the error. when you remove the tag., everything looks good maybe @leandrolanzieri knows why we added this tag. i don't remember.
> 
> The top margin was introduced for the second row, I guess the idea was that the matrix looked not so cluttered. We can add it to hackster or remove it altogether.

ah, yes. @miri64, can you add it to hackster too?

**Comment by miri64** [comment]
See https://github.com/RIOT-OS/riot-os.org/pull/88

**Pull request merged**